charts/hono/templates/hono-adapter-mqtt/hono-adapter-mqtt-vertx-secret.yaml

```diff
@@ -33,6 +33,10 @@ stringData:
         certPath: /etc/hono/cert.pem
         tenantIdleTimeout: {{ .Values.adapters.defaultTenantIdleTimeout | quote }}
         {{- end }}
+        {{- if .Values.adapters.mapperEndpoints }}
```

Comment:
this will always evaluate to `null` FMPOV as the corresponding property is defined at path
`adapters.mqtt.mapperEndpoints` in the values.yaml file. I do not think that we need to define the mapper endpoints per adapter type because we already talked about having this as a general feature available to all adapters, right? So I would rather update the values.yaml file accordingly and move the property right beneath the `adapters` property.
WDYT?

Reply:
good catch. updated the `values.yaml` file